lib/hotel.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+module Administration
+  class Hotel
+
+    attr_reader :all_rooms
+
+    def initialize(rooms)
+      i = 0
+      @all_rooms = Array.new(rooms) {|i| i+1}
+      @all_reservations = []
+    end
+
+    def listing_rooms
+      return @all_rooms
+    end
+
+    def all_reservations
+      @all_reservations = []
```

Comment:
This method seems to blank the list of reservations and then makes a new reservation, but with symbols for arguments.  It doesn't make much sense.  

Then it returns the list (with 1 element).